pydantic/main.py

```diff
@@ -52,6 +54,19 @@
     ConfigType = Type['BaseConfig']
     Model = TypeVar('Model', bound='BaseModel')
 
+    class SchemaExtraCallable(typing_extensions.Protocol):
+        @overload
+        def __call__(self, schema: Dict[str, Any]) -> None:
+            pass
+
+        @overload
+        def __call__(self, schema: Dict[str, Any], model: Type['Model']) -> None:
```

Comment:
Surely you don't need both this case and, the one below?

Better to use `model_class` than `model` to be clear.

You need to add `. # noqa F811` to get linting to pass, see other examples in the code base

Reply:
Done.